src/utils/executeELM.js

```diff
@@ -419,13 +437,14 @@ function updateSearchParams(params, release, type) {
           params.set("_id", INSTRUMENT_LIST.join(","));
           break;
         case "QuestionnaireResponse":
-          params.set("_count", 300);
           params.set("_sort", "_lastUpdated");
           break;
         default:
         // nothing
       }
     }
+    //set default count
+    params.set("_count", 300);
```

Comment:
Do we still need this? Since you were able to fix the url, this seems like a duplicate fix. Am I missing something?

Reply:
thank you @ivan-c. removed.